src/page/home.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState, useEffect } from "react";
+import Todos from "../components/todos";
+import InputForm from "../components/input.form";
+import { nanoid } from "nanoid";
+import { todoType } from "../components/types/todo.type";
+const Home = () => {
+  const initialTodos: todoType[] = JSON.parse(
+    localStorage.getItem("todos") || "[]"
+  );
+  const [todos, setTodos] = useState(initialTodos);
```

Comment:
It is fiixed

Reply:
> Since accessing the local storage and using JSON methods can be an expensive (slow) task. It is recommended to use lazy state initialization and pass a function that returns your data as initial state instead, to save some performance. (this is more relevant for bigger applications but you can still adhere to that good practice)
> 
> You can learn more about Lazy state initialization [HERE](https://kentcdodds.com/blog/use-state-lazy-initialization-and-function-updates)

Did you look into this?